Add @RumScreenName annotation for customization (#261)
signalfx/splunk-otel-android: README.md

| Slow     | >16ms  | slowRenders    | count       | 
| Frozen   | >700ms | frozenRenders  | count       |

> ## Customizing Screen Names

Reviewer: ```suggestion
## Customizing screen names
```

> By default, instrumentation will use the simple class name of each `Fragment`
> and `Activity` type in the `screen.name` attribute. If you wish to customize this
> behavior, you may use the `@RumScreenName` annotation.

Reviewer: ```suggestion
By default, the instrumentation uses the simple class name of each `Fragment`
and `Activity` type as the value of the `screen.name` attribute. To customize the
screen name, use the `@RumScreenName` annotation.
```

> For example, the following Activity will show up with the `screen.name` 
> set to the value "Buttercup".

Reviewer: ```suggestion
For example, the following activity appears with the `screen.name` 
attribute set to the value "Buttercup":
```

```java
@RumScreenName("Buttercup")

> public class MainActivity extends Activity {

Reviewer 2: Nice, I like it way more than the `.setScreenName()` idea 👍
